### DISCUSSION TITLE
I can't get verified!

### DISCUSSION THREAD

**Discussion by ItsToastmeal** [community/community#17731] [GitHub Education]
My school doesn't give us emails (other than ones that block external messages), no student IDs and I only have a digital transcript in Hebrew (aka another language). I tried uploading the original one, a translated one, a screenshot of the school portal, and a translated screenshot, and nothing helps. What do I do???

**Comment by proarafat** [comment]
If you send them your admission papers, they will probably approve you.
like: 1. Admission Congratulation Certificate, 2. Admission Payment Certificate , 3. Etc.....

**Reply by ItsToastmeal** [reply]
I passed the bot check a few hours ago by using a PDF Scanner and a school trip schedule, now I just have to hope I will pass the human check.

**Reply by proarafat** [reply]
yup congrajulation!

**Comment by Zeroshi** [comment]
How are you sending screenshots? I am not given an upload button option. I have the same issue concerning digital files. I have tried both Chrome and Edge browsers.

**Reply by proarafat** [reply]
Cleare your Browser Cach Files
